rfcs/036-holdings-records/README.md

```diff
@@ -0,0 +1,261 @@
+    See [RFC 035](https://github.com/wellcomecollection/docs/tree/master/rfcs/035-marc-856) about how we model electronic locations for more discussion of this field.
+
+*   **949 Shelfmark.**
+    This contains values like /JOU, /HIS and /MED.
+    These are currently exposed on wellcomelibrary.org, e.g. [b13488284](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1348828?lang=eng)
+
+*   **[843 Reproduction Note](https://help.oclc.org/Metadata_Services/Local_Holdings_Maintenance/OCLC_MARC_local_holdings_format_and_standards/8xx_fields/843_Reproduction_Note).**
+    It's not clear how we're actually using this field: it contains values like "Library Display" and "Reading Room", accompanied by a range of dates, sometimes years passed.
+
+We also have a handful of records using fields 000, 561, 596, 852, 860, 876, 990.
+Because these are so infrequently used, we will skip using them in the initial model.
+
+----
+
+
+
+
+
+
+
+## How should we present holdings records in the Catalogue API?
+
+### Can we present holdings records as Items?
+
+The Catalogue API already has a model for `Work` and `Item`.
+The Work model is populated from bibs, and the `Item` model from items.
+
+We did consider trying to put data from holdings records into the `Item` model, but the data in holdings records doesn't really fit.
+For example:
+
+-   Two items titled *Volume 1* and *Volume 2*, and a holdings record titled *Complete set*.
+    It's useful to know that these two volumes form a complete set, but it wouldn't make sense to store this information on the individual items.
+
+-   Two items titled *Volume 1* and *Volume 2*, and a holdings record titled *Missing volume 3*.
+    Here the holdings record is even more important, but there's no way to encode the absence of an item.
+
+-   A run of journals, with each issue itemised, and a holdings record titled *100 issues, 1957–1972*.
+    The holdings record can provide useful summary information that isn't easily observed from the items.
+
+### Creating a separate Holdings type
+
+Rather than reuse the `Item` type, we will create a new `Holdings` type with the same fields (at least, initially).
+This will appear as a list on a Work called `"holdings"`.
+
+We will populate it as follows:
+
+*   The `title` will come from the concatenated contents of MARC field 866 subfields $a and $z.
+    We will append the label from 853 and 863 (newline-separated), one per 853/863 pair.
+    (We might need to split this out if it becomes unwieldy.)
+
+*   The `locations` list will contain:
+
+    *   A digital location using field 856 if the location in fixed field 40 is `elro` (online)
+    *   A physical location using fixed field 40 for the location type/label, and 949 for the shelfmark.
+
+*   For now, we won't add any identifiers.
+    We could add them as h-prefixed Sierra IDs, but I don't see much use of those identifiers -- usually people use the b number when discussing the associated holdings record.
```

Comment:
True, though we do for items, so it possibly makes sense to treat these in the same way.

Reply:
I've added a bit more about this to the RFC.

With items, the identifier to use is unambiguous – add a check digit, add an `i` prefix. With holdings, it's a bit less clear – I've suggested `h` because these are holdings records, but I've seen at least one person talk about these with a `c` prefix for "checkin records".

We also use the item identifier in certain contexts (e.g. to link to a digitised item page), and it's less obvious to me that we'd use a holdings identifier for anything. AIUI, these records are typically looked up in Sierra by the bib number, not the holdings number.

I'm not opposed to adding these identifiers if there's a good use case, and we can nail down the prefix, but my default position is to omit them in v1.

Reply:
Certainly wouldn't want to invent a prefix, so let's leave out if we're not sure.